### PR TITLE
Build Lunar on the Xenial test farm

### PR DESCRIPTION
**Pull request by nuclearsandwich**
⚠️ Not intended for merge. This PR is just to conveniently display and discuss the changes.

Based on the `production` branch because master appears out of date (Lunar isn't represented) and this is a pre-production test.

@mikaelarguedas have I correctly excised anything that would make noise for current ROS maintainers?

@tfoote is there anything else that would compromise the production infrastructure here? The puppet / hiera config this will be run on is at https://github.com/nuclearsandwich/bf_config_experiments/tree/xenialize

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -2,52 +2,12 @@
...
-    source_builds:
-      default: kinetic/source-build.yaml
   lunar:
     doc_builds:
       default: lunar/doc-build.yaml
       released-packages-without-doc-job: lunar/doc-released-build.yaml
     notification_emails:
-    - ros-buildfarm-lunar@googlegroups.com
-    - mikael+buildfarm@osrfoundation.org
+    - steven@openrobotics.org
```
Comment (dirk-thomas) [inline index.yaml:10]: Maybe consider an email like `steven+buildfarm@openrobotics.org` to ease of filtering.

**Review by dirk-thomas — COMMENTED** [review]
The config looks good to me.

**Comment by mikaelarguedas** [comment]
> @mikaelarguedas have I correctly excised anything that would make noise for current ROS maintainers?

:+1: lgtm

**Comment by tfoote** [comment]
This looks reasonable to me.

**Comment by nuclearsandwich** [comment]
Closing since this was only ever intended to get review on a test configuration.

**Pull request closed without merging**